bim2sim/tasks/bps/sb_correction.py

```diff
@@ -423,6 +428,8 @@ def create_new_convex_bounds(self, convex_shapes: list[TopoDS_Shape],
                         opening.parent_bound = new_bound
             # check and fix surface normal if needed
             if not all([abs(i) < 1e-3 for i in (
+                    # TODO bound normal is delete in function call in line 409 and does
+                    #  not exist here. Why check? this fails with new attributes
```

Comment:
Is this still valid and need to be fixed or already addressed in the latest changes?

Reply:
This ensures the matching surface normal orientation of the new created bound (same normal direction as previous shape). This is probably currently not properly tested, but caused issues before, so I would keep this check here.